src/model.rs

```diff
@@ -1448,7 +1389,7 @@ impl Model {
         )?;
 
         c.update_stop_area_coords();
-        c.enhance_with_co2();
+        fill_co2(&mut c);
```

Comment:
I thought you would keep this here.
```suggestion
        enhancers::fill_co2(&mut c);
```

Reply:
you're right, I changed it (but did not took the commit to also change the import)